tests/caasp/stack_cap.pm

```diff
@@ -0,0 +1,93 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Deploy CAP and run test app
+# Maintainer: Martin Kravec <mkravec@suse.com>
+
+use parent 'caasp_controller';
+use caasp_controller;
+
+use strict;
+use testapi;
+use caasp 'script_retry';
+use utils qw(zypper_call systemctl);
+
+sub run {
+    switch_to 'xterm';
+
+    my $mip = script_output q#host -cIN master-api.openqa.test | awk '{print $NF}'#;
+    my $xip = "$mip.xip.io";
+
+    become_root;
+    systemctl 'stop SuSEfirewall2';
```

Comment:
So am I right that there is no firewalld in any version of caasp, where this test will be used?

Reply:
right, we have sles 12-sp3 support-server and plan is to upgrade to 12-sp4 next year

Reply:
Alright, I personally would write this to have a chance not to touch this code once we upgrade, but it's still a probability =)